assets/out

```diff
@@ -131,7 +131,18 @@ elif [ -n "$build" ]; then
     expanded_build_args=${expanded_build_args}" "$(jq -r 'with_entries(.key |= "--build-arg " + . )|with_entries(.key = .key + "=" +.value)|keys|join(" ")' <$build_args_file)
   fi
 
-  docker build -t "${repository}:${tag_name}" $expanded_build_args -f "$dockerfile" "$build"
+  already_built=false
+  if [ "$if_not_exists" = "true" ]; then
```

Comment:
The `if_not_exists` was never assigned/parsed from the JSON payload, so it will always be undefined. Right